# HG changeset patch
# User Rik <rik@octave.org>
# Date 1528924848 25200
#      Wed Jun 13 14:20:48 2018 -0700
# Branch stable
# Node ID e0906c38f18e3ae24a80b483ef028787fe5b2ca7
# Parent  f09f7c7815e3b7713ced2d5e17dff54b1de16e0f
doc: Use @var rather than @code to mark third argument to sort in docstring.

* data.cc (Fsort): Use @var rather than @code to mark third argument (mode).

diff --git a/libinterp/corefcn/data.cc b/libinterp/corefcn/data.cc
--- a/libinterp/corefcn/data.cc
+++ b/libinterp/corefcn/data.cc
@@ -6387,19 +6387,19 @@ For example:
 sort ([1, 2; 2, 3; 3, 1])
    @result{}  1  1
        2  2
        3  3
 @end group
 @end example
 
 If the optional argument @var{dim} is given, then the matrix is sorted
-along the dimension defined by @var{dim}.  The optional argument @code{mode}
+along the dimension defined by @var{dim}.  The optional argument @var{mode}
 defines the order in which the values will be sorted.  Valid values of
-@code{mode} are @qcode{"ascend"} or @qcode{"descend"}.
+@var{mode} are @qcode{"ascend"} or @qcode{"descend"}.
 
 The @code{sort} function may also be used to produce a matrix
 containing the original row indices of the elements in the sorted
 matrix.  For example:
 
 @example
 @group
 [s, i] = sort ([1, 2; 2, 3; 3, 1])
